helloworld/src/main/java/FooBar.java

```diff
@@ -1,6 +1,7 @@
 public class FooBar {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class should define a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.coding.MissingCtorCheck)